crates/db_schema/src/impls/post.rs

```diff
@@ -160,6 +161,10 @@ impl Post {
       update = update.filter(post::community_id.eq(for_community_id));
     }
 
+    if let Some(for_instance_id) = for_instance_id {
+      update = update.filter(post::instance_id.eq(for_instance_id));
+    }
```

Comment:
`post.instance_id` is the instance where the community is hosted, right? Otherwise this would be wrong. Not entirely clear to me where this gets set.

Reply:
Yikes, it looks like instance_id isn't being set (and its default 0!). 

I'll make an issue to remove that column, because it should rely on the community's `instance_id` anyway.